init.go

```diff
@@ -46,33 +56,36 @@ func main() {
 		reportCommand.Parse(os.Args[2:])
 	default:
 		fmt.Println(reportUsageMessage)
-		os.Exit(1)
+		return 1
 	}
 
 	// Get current path
 	currentDir, err := os.Getwd()
 	if err != nil {
 		fmt.Println("DeepSource | Error | Unable to identify current directory.")
-		os.Exit(0)
+		sentry.CaptureException(err)
```

Comment:
How would we know more about the user? Log the DEEPSOURCE_DSN and the args in sentry, too.

Reply:
Nice catch. Done!